### PR TITLE
raftstore: only print CommitMerge once

### PR DESCRIPTION
**Pull request by BusyJay**
## What have you changed? (mandatory)

Make the commit merge log only be printed once. In the new mechanism, commit merge will be executed twice. So the log will be printed twice. I move the log into the place where commit merge is actually happen to make it only be committed once.

## What are the type of the changes? (mandatory)

- Improvement

## How has this PR been tested? (mandatory)

Manual tests

## Does this PR affect documentation (docs) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

File: src/raftstore/store/fsm/apply.rs [hunk 1]
```diff
@@ -1731,6 +1731,14 @@ impl ApplyDelegate {
             ));
         }
 
+        info!(
+            "{} execute admin command {:?} at [term: {}, index: {}]",
+            self.tag,
+            req,
+            ctx.exec_ctx.as_ref().unwrap().term,
+            ctx.exec_ctx.as_ref().unwrap().index;
```
Comment (BusyJay) [inline src/raftstore/store/fsm/apply.rs]: Oh, can't believe that `;` will work here.

Comment (overvenus) [reply src/raftstore/store/fsm/apply.rs]: Yeah, macros from slog supports this.

File: src/raftstore/store/fsm/apply.rs [hunk 2]
```diff
@@ -1004,13 +1004,13 @@ impl ApplyDelegate {
...
         let cmd_type = request.get_cmd_type();
-        if cmd_type != AdminCmdType::CompactLog {
+        if cmd_type != AdminCmdType::CompactLog && cmd_type != AdminCmdType::CommitMerge {
             info!(
                 "{} execute admin command {:?} at [term: {}, index: {}]",
                 self.tag,
                 request,
                 ctx.exec_ctx.as_ref().unwrap().term,
-                ctx.exec_ctx.as_ref().unwrap().index
+                ctx.exec_ctx.as_ref().unwrap().index;
```
Comment (siddontang) [inline src/raftstore/store/fsm/apply.rs]: em, why add ';' here?

File: src/raftstore/store/fsm/apply.rs (hunk 1 above)
Comment (zhangjinpeng1987) [reply src/raftstore/store/fsm/apply.rs]: But the `;` is not necessary here, please remove it.

**Pull request merged**